source-repo-scripts/bump_dependency.bash

```diff
@@ -250,7 +250,6 @@ for ((i = 0; i < "${#LIBRARIES[@]}"; i++)); do
       echo -e "${RED}No main or master branch found on ${YAML_FILE}.${DEFAULT}"
       exit
     fi
-  fi
```

Comment:
probably need to keep this `fi`

Reply:
ouch! thanks [0917305](https://github.com/gazebo-tooling/release-tools/pull/723/commits/0917305505610327fbe7d22b4fffbb17d9596a82)